Makefile

```diff
@@ -61,6 +63,8 @@ checkformat:
 
 clean-gentype:
 	make -C jscomp/gentype_tests/typescript-react-example clean
+	make -C jscomp/gentype_tests/typescript-res-node16 clean
```

Comment:
Do we need this or does it only test the different extensions?
If all we want to know is that the extensions are generated, I would avoid adding 2 new subproject only for that. But perhaps this tests certain compatibility assumptions?

Reply:
They have different `moduleResolution` setting in the `tsconfig.json`. It checks compatibility by inputting the generated files to tsc.

Reply:
Sure but in practice, what does it check other than the printing function prints the right extension for the 3 cases?
If that's all, the tests should be dropped as they will never catch any bugs, and add up dev time and CI time.

Reply:
Would it be better to delete these projects completely?

Reply:
Yes unless there's something they demonstrate of independent value to ReScript users.